0017-negotiation-and-execution-protocol.adoc

```diff
@@ -0,0 +1,544 @@
+= How to document cnd's HTTP API =
+Thomas Eizinger <thomas@coblox.tech>; Philipp Hoenisch <philipp@coblox.tech>; Daniel Karzel <daniel.karzel@coblox.tech>;
+:toc:
+:revdate: 2019-10-05
+:sectnums:
+:sectnumlevels: 5
+
+NOTE: Author: {authors} +
+Date: {revdate} +
+Tracking issue: https://github.com/comit-network/comit-rs/issues/1413[#1413]
+
+== Context ==
+
+Over time we came to the conclusion that our protocol stack in general, and RFC003 in particular, is difficult to understand in certain ways.
+
+We could specify several problems:
+
+. The Accept/Decline feels weird, it often does not make sense in specific use-case context
+. We do not clearly distinguish trade negotiation from execution at the moment.
+. It is hard to add the concept of orders and an order-book to COMIT.
+
+This spike includes:
+
+. Solution proposal for splitting trade negotiation and execution in different protocols.
+. Solution proposal for the "How to update an offer"/"Offers should not get stale" problem.
```

Comment:
Suggest using the term 'order' since that is the term used when trading equities or forex.  Then in this context orders car be cancelled and new orders issued.